shared/common-adapters/header-hoc.native.js

```diff
@@ -18,7 +18,7 @@ function HeaderHoc<P> (WrappedComponent: ReactClass<P>) {
         {onCancel && <Text type='BodyBigLink' onClick={onCancel}>Cancel</Text>}
         {onBack && <BackButton iconStyle={_backButtonIconStyle} onClick={onBack} />}
       </Box>
-      {<WrappedComponent {...restProps} />}
+      {<WrappedComponent {...restProps} onBack={onBack} onCancel={onCancel} />}
```

Comment:
the header hoc shouldn't stop these from going into the component. Sometimes we'll have a button that does an onBack or onCancel and using the hoc eats it before.